src/common_subset.rs

```diff
@@ -7,22 +7,41 @@ use std::collections::{HashMap, HashSet, VecDeque};
 use std::fmt::{Debug, Display};
 use std::hash::Hash;
 
-use agreement;
-use agreement::{Agreement, AgreementMessage};
-
-use broadcast;
-use broadcast::{Broadcast, BroadcastMessage};
-
+use agreement::{self, Agreement, AgreementMessage};
+use broadcast::{self, Broadcast, BroadcastMessage};
 use messaging::{Target, TargetedMessage};
 
 // TODO: Make this a generic argument of `Broadcast`.
 type ProposedValue = Vec<u8>;
+
 // Type of output from the Common Subset message handler.
 type CommonSubsetOutput<NodeUid> = (
     Option<HashSet<ProposedValue>>,
     VecDeque<TargetedMessage<Message<NodeUid>, NodeUid>>,
 );
 
+#[derive(Clone, Debug)]
```

Comment:
Is there a rule in your coding standard for `Error` to be moved over here? You mentioned only imports and namespace operators in the PR description. Does this change make code any better?

Reply:
But this change is a totally subjective, and is one that I don't feel strongly about. If you have a preference one way or another,  I can change it back without a problem.